lib_bgp_data/cdn_whitelist/tests/test_cdn_whitelist.py

```diff
@@ -27,11 +29,26 @@ class Test_CDN_Whitelist:
     @pytest.fixture(autouse=True)
     def setup(self):
         """Parser setup and table deleted before every test"""
-
-        pass
+        with CDN_Whitelist_Table(clear=True) as t:
+            pass
 
     def test_run(self):
         """Tests the _run function"""
 
         # Downloads data and inserts into the database (should be more than 0 entries)
         # Makes sure it doesn't error even if there is a CDN that doesn't exist
+
+        # Add a nonsense CDN
+        list_path = path.dirname(path.dirname(path.realpath(__file__)))
+        list_path = path.join(list_path, 'cdns.txt')
+        utils.run_cmds(f'echo "urfgurf" >> {list_path}')
```

Comment:
If your test failed in the middle of it, would your file would be permanently changed? Perhaps a patch here would work better?